packages/nodes-base/nodes/SpreadsheetFile/test/SpreadsheetFile.test.ts

```diff
@@ -87,9 +87,9 @@ describe('Execute Spreadsheet File Node', () => {
 										mimeType: 'text/csv',
 										fileType: 'text',
 										fileExtension: 'csv',
-										data: '77u/QSxCLEMKMSwyLDMKNCw1LDYK',
+										data: '77u/QSxCLEMKMSwyLDMKNCw1LDY=',
```

Comment:
had to update this test because the generated csv files don't have an additional empty line at the end of the file anymore.